Appraisals

```diff
@@ -587,6 +587,15 @@ elsif ruby_version?('2.3')
   appraise 'core-old' do
     gem 'dogstatsd-ruby', '~> 4'
   end
+
+  # Somehow, I failed to install this appraisal group with Ruby 2.4
```

Comment:
Should this comment be here?

Reply:
I supposed we are going to deprecate old ruby and Rails 3.x in the future, so did not invest too much solving this, so I left a comment instead.